### ISSUE TITLE
The training speed is very slow

### ISSUE THREAD

**Issue by Huozhiyuan** [WenjieDu/PyPOTS#455]
### 1. System Info

python 3.9.19

### 2. Information

- [X] The official example scripts
- [ ] My own created scripts

### 3. Reproduction

from pypots.imputation import SAITS
from pypots.utils.metrics import calc_mae
saits = SAITS(n_steps=48, n_features=37, n_layers=2, d_model=256, d_ffn=128, n_heads=4, d_k=64, d_v=64, dropout=0.1, epochs=10)
# Here I use the whole dataset as the training set because ground truth is not visible to the model, you can also split it into train/val/test sets
saits.fit(dataset)  # train the model on the dataset

![image](https://github.com/WenjieDu/PyPOTS/assets/167734466/a89de794-61db-4053-b6bb-1f6de97cad38)

### 4. Expected behavior

Hi everyone:
I'm a beginner in deep learning，when I was running the official example of the SAITS model with pypots and physionet_2012 dataset,' saits.fit(dataset)' was running very slowly and took all night.  I read in another issues that training is usually no more than two hours, so how can i do ?

**Comment by Huozhiyuan** [comment]
您的邮件我已收到，会尽快回复

**Comment by WenjieDu** [comment]
@Huozhiyuan Use CUDA for speeding up.

**Comment by Huozhiyuan** [comment]
thanks a lot, i will try
